tutorials/00-introduction/00_introduction.jmd

```diff
@@ -8,92 +8,100 @@ redirect_from: tutorials/0-introduction/
 
-# Here's what the first five coin flips look like:
+Here is what the first five coin flips look like:
+
+```julia
 data[1:5]
 ```
 
-After flipping all our coins, we want to set a prior belief about what we think the distribution of coin flips look like. In this case, we are going to choose a common prior distribution called the [Beta](https://en.wikipedia.org/wiki/Beta_distribution) distribution.
+Next, we specify a prior belief about the distribution of heads and tails in a coin toss.
+Here we choose a [Beta](https://en.wikipedia.org/wiki/Beta_distribution) distribution as prior distribution for the probability of heads.
+Before any coin flip is observed, we assume a uniform distribution $\operatorname{U}(0, 1) = \operatorname{Beta}(1, 1)$ of the probability of heads.
+I.e., every probability is equally likely initially.
 
 ```julia
-# Our prior belief about the probability of heads in a coin toss.
 prior_belief = Beta(1, 1);
 ```
 
 With our priors set and our data at hand, we can perform Bayesian inference.
 
-This is a fairly simple process. We expose one additional coin flip to our model every iteration, such that the first run only sees the first coin flip, while the last iteration sees all the coin flips. Then, we set the `updated_belief` variable to an updated version of the original Beta distribution that accounts for the new proportion of heads and tails.
-
-For the mathematically inclined, the `Beta` distribution is updated by adding each coin flip to the distribution's $\alpha$ and $\beta$ parameters, which are initially defined as $\alpha = 1, \beta = 1$. Over time, with more and more coin flips, $\alpha$ and $\beta$ will be approximately equal to each other as we are equally likely to flip a heads or a tails, and the plot of the beta distribution will become more tightly centered around 0.5.
-
-This works because mean of the `Beta` distribution is defined as the following:
-
-$$\text{E}[\text{Beta}] = \dfrac{\alpha}{\alpha+\beta}$$
-
-Which is 0.5 when $\alpha = \beta$, as we expect for a large enough number of coin flips. As we increase the number of samples, our variance will also decrease, such that the distribution will reflect less uncertainty about the probability of receiving a heads. The definition of the variance for the `Beta` distribution is the following:
-
-$$\text{var}[\text{Beta}] = \dfrac{\alpha\beta}{(\alpha + \beta)^2 (\alpha + \beta + 1)}$$
-
-The intuition about this definition is that the variance of the distribution will approach 0 with more and more samples, as the denominator will grow faster than will the numerator. More samples means less variance.
+This is a fairly simple process.
+We expose one additional coin flip to our model every iteration, such that the first run only sees the first coin flip, while the last iteration sees all the coin flips.
+In each iteration we update our belief to an updated version of the original Beta distribution that accounts for the new proportion of heads and tails.
```

Comment:
```suggestion
In each iteration we update our belief to an updated version of the original Beta distribution that accounts for the new proportion of heads and tails.
The update is particularly simple since our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior).
Note that a closed-form expression for the posterior (implemented in the `updated_belief` expression below) is not accessible in general and usually does not exist for more interesting models.
```